src/cf.data.pre

```diff
@@ -5462,11 +5479,13 @@ TYPE: HelperChildConfig
 DEFAULT: 20 startup=0 idle=1 concurrency=0
 LOC: Config.redirectChildren
 DOC_START
-	The maximum number of redirector processes to spawn. If you limit
-	it too few Squid will have to wait for them to process a backlog of
-	URLs, slowing it down. If you allow too many they will use RAM
-	and other system resources noticably.
-	
+	Specifies the maximum number of redirector processes that Squid may
+	spawn (numberofchildren) and several related options. Using too few
+	helper processes queues requests. Using too many helpers wastes RAM
```

Comment:
It should be called either "helper processes" or just "helpers".

Reply:
Please see if you like the rephrased version better.